jwst/cube_skymatch/cube_skymatch_step.py

```diff
@@ -169,10 +167,10 @@ def _reset_background(self, models):
             del m.meta.background
 
     def _set_cube_bkg_meta(self, model, sky_cube):
-        np = len(meta.background.polynomial_info)
-        if np > 1 or (np == 1 and meta.background.polynomial_info[0].channel):
+        np = len(model.meta.background.polynomial_info)
```

Comment:
I am assuming the intention here (and elsewhere) was to reference `model.meta` as `meta` is not defined.

Reply:
It's a good assumption.